app/api/helpers/mail.py

```diff
@@ -66,7 +66,7 @@ def send_email(to, action, subject, html):
                     "Content-Type": "application/json"
                 }
                 from .tasks import send_email_task
-                send_email_task.delay(payload, headers)
+                send_email_task(payload, headers)
```

Comment:
Will this change sending the email from asynchronously to synchronously?

Reply:
Yes it sends the mail immediately. 
.delay wasn't working actually.

Reply:
Can we first debug the `delay` method? It would be better to keep this feature async.

Reply:
That's what I meant. This is a workaround, not a fix

Reply:
Fair point. But for now we want to make it run as less than a week left for jugaadfest. No events canbe published due to this issue.

Reply:
Add a TODO comment and open a new issue regarding fixing this